clients/src/main/java/org/apache/kafka/clients/consumer/ConsumerRebalanceListener.java

```diff
@@ -178,7 +178,7 @@ public interface ConsumerRebalanceListener {
      * For example, this function is called if a consumer's session timeout has expired, or if a fatal error has been
      * received indicating the consumer is no longer part of the group.
      * <p>
-     * By default it will just trigger {@link ConsumerRebalanceListener#onPartitionsRevoked}; for users who want to distinguish
+     * By default, it just triggers {@link ConsumerRebalanceListener#onPartitionsRevoked}; for users who want to distinguish
```

Comment:
I'm not a grammar expert, but what about:

```suggestion
     * By default this function triggers {@link ConsumerRebalanceListener#onPartitionsRevoked}; for users who want to distinguish
```